code/modules/surgery/organs/lungs.dm

```diff
@@ -102,28 +102,16 @@
 			H.throw_alert("not_enough_nitro", /atom/movable/screen/alert/not_enough_nitro)
 		return FALSE
 
+	for(var/gas_id in GLOB.meta_gas_info)
+		breath.assert_gas(gas_id)
```

Comment:
Couldn't you call `assert_gases(GLOB.meta_gas_info)`?

Reply:
right, doing that

Reply:
uh no not doing that, apparently it runtimes on gas code saying that assert gas can't pass null.copy()

Reply:
that's very strange, could you send me the branch you used? that implies that meta_gas_info was at some point null when a carbon breathed, which should be impossible

Reply:
https://github.com/Ghilker/tgstation/tree/filtering_masks yeah i know i think is strange too

Reply:
I meant like, the code that caused the error. I'd like to poke at it a bit

Reply:
?